### PR TITLE
Es 1.3.0

### PR DESCRIPTION
**Pull request by soniar4i**
Spanish translates for 1.3.0 Sass guidelines are ready

### CODE REVIEW

**Comment by HugoGiraudel** [comment]
That’s awesooooome! 
I’ll check that during the week, thank you so much. :)

**Comment by soniar4i** [comment]
yes!! I finally found time to update it  ;)

Thanks to you for being patient!

Let me know if something is missing and I'll change it asap.

2017-03-12 23:58 GMT+01:00 Hugo Giraudel <notifications@github.com>:

**Comment by HugoGiraudel** [comment]
I did a first run and it looks good! I’ll do a proper check up during the week. :)

**Comment by soniar4i** [comment]
Great! 🙌🎉

2017-03-14 20:48 GMT+01:00 Hugo Giraudel <notifications@github.com>:

File: pages/es/_loops.md [hunk 1]
```diff
@@ -17,22 +17,23 @@ Cuando se itera sobre un mapa utiliza siempre `$key` y `$value` como nombres de
...
 
 {% include snippets/loops/03/index.html %}
 
 Utiliza siempre `$i` como nombre de variable para mantener la convención habitual, a menos que tengas una muy buena razón para no hacerlo, no uses la palabra clave `to`, es mejor usar siempre `through`. Muchos desarrolladores no saben que Sass ofrece esta variación; usarla podría llevar a confusión.
 
 También asegúrate de respetar estas directrices para preservar la legibilidad:
 
-* Deja siempre una línea en blanco antes de `@each`;
+* Deja siempre una línea en blanco antes del `@for`;
 * DEja siempre una línea en blanco después de la llave de cierre (`}`) a no ser que la siguiente línea sea otra llave de cierre (`}`).
```
Comment (HugoGiraudel) [inline pages/es/_loops.md]: Deja* ;)

File: pages/es/_syntax.md [hunk 2]
```diff
@@ -59,22 +61,17 @@ Si una cadena contiene una o varias comillas simples, se podría considerar envo
...
-
 ## Números
 
-En Sass, el número es un tipo de datos que incluye de todo, desde números sin unidades a longitudes, pasando por duraciones, frecuencias y ángulos, entre otros. Esto permite que se puedan realizar cálculos en estas medidas.
+En Sass, número es un tipo de datos que incluye de todo, desde números sin unidades a longitudes, pasando por duraciones, frecuencias y ángulos, entre otros. Esto permite que se ejecuten cálculos sobre tales medidas.
 
 ### Ceros
 
-Siempre se deben mostrar los ceros a la izquierda antes de un valor decimal menor que uno. Nunca mostrar ceros finales.
+Siempre se deben mostrar los ceros a la izquierda antes de un valor decimal menor que uno. Nunca mostrar los ceros finales.
```
Comment (HugoGiraudel) [inline pages/es/_syntax.md]: There is a paragraph missing here. See English version for reference: https://github.com/HugoGiraudel/sass-guidelines/pull/223/files#diff-8fb504ae90cfe32f3ba366da9f47f1ddR85.

**Comment by HugoGiraudel** [comment]
Alright, I have been going through it and it is almost perfect! Everything looks to be in order; only 2 small paragraphs are missing. Do you feel like you could add them @soniar4i? :)

**Comment by soniar4i** [comment]
Ok! I'm on it right now!

😊

**Comment by soniar4i** [comment]
Ok! @HugoGiraudel I think it's done. Let me know if something is still missing :)

**Comment by HugoGiraudel** [comment]
Thank you so much for your work @soniar4i! :)

**Comment by HugoGiraudel** [comment]
@soniar4i If you have a second to take a look at this: we have some broken internal links in the Spanish version.

screen shot 2017-03-20 at 23 16 38

**Comment by HugoGiraudel** [comment]
No rush. :)

**Comment by soniar4i** [comment]
By the way, it's possible this could be related with accents... how can i
deal with them when they appear as the title of a section?

2017-03-21 8:43 GMT+01:00 Hugo Giraudel <notifications@github.com>:

> No rush. :)
>

**Comment by HugoGiraudel** [comment]
Let me have a look.

**Comment by HugoGiraudel** [comment]
Right, so the Markdown parser removes them from the hashes. *CÁLCULOS* becomes `#clculos`. I’ll see if I can fix these. :)

**Comment by HugoGiraudel** [comment]
Alright, emitted a fix: https://github.com/HugoGiraudel/sass-guidelines/commit/4a7eeb74320bcfbede413a07dacee584394bef0f. Should be alright. :)

**Comment by soniar4i** [comment]
So it should be good now, or do i have to change something?


2017-03-22 9:55 GMT+01:00 Hugo Giraudel <notifications@github.com>:

**Comment by HugoGiraudel** [comment]
It’s all good, thank you!

**Comment by soniar4i** [comment]
Thanks to you!

2017-03-22 10:02 GMT+01:00 Hugo Giraudel <notifications@github.com>:

**Pull request merged**